app/helpers/questions_helper.rb

```diff
@@ -0,0 +1,2 @@
+module QuestionsHelper
+end
```

Comment:
Don't really need this file. I think this gets generated if you do rails generate